plugin.go

```diff
@@ -173,20 +173,25 @@ func (p *Plugin) vaultPass() error {
 }
 
 func (p *Plugin) playbooks() error {
-	var playbooks []string
+	var (
```

Comment:
why expand this for single local variable definition?

Reply:
Because it's done with that schema on all plugins.

Reply:
For global variables yes but at least from drone code func local variables are defined as `var xxx ..`

Reply:
Not within the plugin org, I have written that like it on every plugin that I have authored.

Reply:
oh, ok, I was just asking ;)